Fix documentation for Secret Stores query parameters. (#3593)
dapr/docs: daprdocs/content/en/reference/components-reference/supported-secret-stores/alicloud-oos-parameter-store.md

| accessKeySecret    | Y        | The AlibabaCloud Access Key Secret to access this resource              | `"accessKeySecret"`  |
| securityToken      | N        | The AlibabaCloud Security Token to use                                  | `"securityToken"`    |

## Optional per-request metadata properties

> The following [optional query parameters]({{< ref "secrets_api#query-parameters" >}}) can be provided when retrieving secrets from this secret store:

Reviewer: ```suggestion
The following [optional query parameters]({{< ref "secrets_api.md#query-parameters" >}}) can be provided when retrieving secrets from this secret store:
```

Query Parameter | Description
--------- | -----------

> `metadata.version_id` | version for the given secret key

Reviewer: ```suggestion
`metadata.version_id` | Version for the given secret key
```

> `metadata.path` | (For bulk requests only) the path from the metadata. If not set means root path (all secrets).

Reviewer: ```suggestion
`metadata.path` | (For bulk requests only) The path from the metadata. If not set, defaults to root path (all secrets).
```
